game-server/data/handlers/ai/instance/empyreanCrucible/RM1337AI.java

```diff
@@ -59,99 +59,64 @@ public void handleBackHome() {
 	@Override
 	public void handleAttack(Creature creature) {
 		super.handleAttack(creature);
-		if (isHome.compareAndSet(true, false)) {
+		if (isHome.compareAndSet(true, false))
 			startSkillTask1();
-		}
-		if (getLifeStats().getHpPercentage() <= 75) {
-			if (isEventStarted.compareAndSet(false, true)) {
-				startSkillTask2();
+		handleHpPhase(getLifeStats().getHpPercentage());
+	}
+
+	@Override
+	public void handleHpPhase(int phaseHpPercent) {
+		switch (phaseHpPercent) {
+			case 15, 35, 55, 75, 95 -> {
+				if (isEventStarted.compareAndSet(false, true))
+					startSkillTask2();
```

Comment:
I don't understand how this is supposed to behave. `task2` is a recurring task, so you could remove all other percentages except 95 and it would work the same (don't forget to also remove them in the HpPhases constructor).
Also `isEventStarted` can be removed after properly implementing the HpPhases handler (see my other comments).